app/views/PageParents.vue

```diff
@@ -1,28 +1,475 @@
+                                         /images/pages/parents/beginner_concepts@2x.png 2x,
+                                         /images/pages/parents/beginner_concepts@3x.png 3x"
+                                 src="/images/pages/parents/beginner_concepts@1x.png"
+                            />
+                        </div>
+                        <h3>{{ $t('parents_landing_2.beginner_concepts') }}</h3>
+                        <ul>
+                            <li>{{ $t('parents_landing_2.beginner_concepts_1') }}</li>
+                            <li>{{ $t('parents_landing_2.beginner_concepts_2') }}</li>
+                            <li>{{ $t('parents_landing_2.beginner_concepts_3') }}</li>
+                            <li>{{ $t('parents_landing_2.beginner_concepts_4') }}</li>
+                            <li>{{ $t('parents_landing_2.beginner_concepts_5') }}</li>
+                            <li>{{ $t('parents_landing_2.beginner_concepts_6') }}</li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="col-lg-4 col-md-4 col-sm-12 col-xs-12">
+                    <div class="concept-overview">
+                        <div class="image-container">
+                            <img srcset="/images/pages/parents/intermediate_concepts@1x.png 1x,
+                                         /images/pages/parents/intermediate_concepts@2x.png 2x,
+                                         /images/pages/parents/intermediate_concepts@3x.png 3x"
+                                 src="/images/pages/parents/intermediate_concepts@1x.png"
+                            />
+                        </div>
+                        <h3>{{ $t('parents_landing_2.intermediate_concepts') }}</h3>
+                        <ul>
+                            <li>{{ $t('parents_landing_2.intermediate_concepts_1') }}</li>
+                            <li>{{ $t('parents_landing_2.intermediate_concepts_2') }}</li>
+                            <li>{{ $t('parents_landing_2.intermediate_concepts_3') }}</li>
+                            <li>{{ $t('parents_landing_2.intermediate_concepts_4') }}</li>
+                            <li>{{ $t('parents_landing_2.intermediate_concepts_5') }}</li>
+                            <li>{{ $t('parents_landing_2.intermediate_concepts_6') }}</li>
+                        </ul>
+                    </div>
+                </div>
 
-        Products Loading: {{ productsLoading }}
-        <div v-if="!productsLoading && basicSubscriptionForCurrentUser">
-            Basic subscription: {{ basicSubscriptionForCurrentUser.amount / 100 }}
+                <div class="col-lg-4 col-md-4 col-sm-12 col-xs-12">
+                    <div class="concept-overview">
+                        <div class="image-container">
+                            <img srcset="/images/pages/parents/advanced_concepts@1x.png 1x,
```

Comment:
You are wild! This is insane and amazing. I love this in so many ways haha!

Wondering on your thoughts of using the biggest images, and then using Dexecure service worker to optimize the image in a similar way. Does this provide a bigger advantage to better usage of Dexecure?

Reply:
Oh interesting does Dexecure automatically do this for us?  Could I just ship the 3x image and in production Dexecure will update to the proper size?

Reply:
I'll include whatever changes come out of this conversation later this week too.

Reply:
I believe so: - You can turn it off with a query param.

Reference: https://support.dexecure.com/hc/url-configuration/disable-auto-resizing-of-images?searched=true

There's also other query params for explicit optimizations (done with client side query param hints):
https://support.dexecure.com/hc/url-configuration